boards/shields/adafruit_winc1500/doc/index.rst

```diff
@@ -0,0 +1,70 @@
+.. _adafruit_winc1500:
+
+Adafruit WINC1500 Wifi Shield
+#############################
+
+Overview
+********
+
+The Adafruit WINC1500 Wifi shield is an Arduino
+compatible shield based on the ATWINC1500 wifi controller
+from Microchip.
+Besides the wifi chip it also contains an SD card reader.
+
+The SD card reader is currently not supported
+
+More information about the shield can be found
+at the `Adafruit WINC1500 website`_.
+
+Pins Assignment of the Adafruit WINC1500 WiFi Shield
+====================================================
+
+
++-----------------------+---------------------------------------------+
+| Shield Connector Pin  | Function                                    |
++=======================+=============================================+
+| D4                    | MicroSD SPI CSn  (Not supported)            |
++-----------------------+---------------------------------------------+
+| D5                    | WINC1500 RST     (/Reset of winc1500)       |
++-----------------------+---------------------------------------------+
+| D6 (b)                | WINC1500 EN      (Enable of winc1500)       |
++-----------------------+---------------------------------------------+
+| D7                    | WINC1500 IRQ     (IRQ from winc1500)        |
++-----------------------+---------------------------------------------+
+| D10                   | WINC1500 SPI CSn                            |
++-----------------------+---------------------------------------------+
+| D11 (a)               | SPI MOSI         (Serial Data Input)        |
++-----------------------+---------------------------------------------+
+| D12 (a)               | SPI MISO         (Serial Data Out)          |
++-----------------------+---------------------------------------------+
+| D13 (a)               | SPI SCK          (Serial Clock Input)       |
++-----------------------+---------------------------------------------+
+
+The (a) marked pins must be jumpered to SPI at the SHIELD
```

Comment:
"The pins marked (a) must be jumpered to the SPI port at the shield."

Reply:
Text changed